### ISSUE TITLE
[RFE] warn about unused Pylint disablings

### ISSUE THREAD

**Issue by stanislavlevin** [PyCQA/pylint#4757]
### Current problem

The project which actively uses Pylint can contain many disabling of Pylint checkers via the code comments:
http://pylint.pycqa.org/en/latest/user_guide/message-control.html
- this can significantly litter a code with tons of comments (Pylint, flake, mypy, etc., individually or all together)
- this can hide actual code errors
- slows down Pylint analysis

### Desired solution

In my opinion, people disable checkers  via the code comments in the next common cases:
- unsupported Pylint feature
- broken Pylint feature (for example, regression after upgrade)

I think it will be useful to have an option to check such disabling and report if disabling is no longer needed (only code comments).

Inspired by mypy's:
> --warn-unused-ignores
This flag will make mypy report an error whenever your code uses a # type: ignore comment on a line that is not actually generating an error message.
>
> This flag, along with the --warn-redundant-casts flag, are both particularly useful when you are upgrading mypy. Previously, you may have needed to add casts or # type: ignore annotations to work around bugs in mypy or missing stubs for 3rd party libraries.
>
> These two flags let you discover cases where either workarounds are no longer necessary.

https://mypy.readthedocs.io/en/stable/command_line.html?highlight=unused-ignores#cmdoption-mypy-warn-unused-ignores

**Comment by Pierre-Sassoulas** [comment]
This already exists you need ``enable=useless-supression`` in the configuration. Maybe it should become a default or be more visible in the documentation, what do you think ?

**Comment by stanislavlevin** [comment]
Note: double `p` in the name (`useless-suppression`) ;-)
It seems that this is exactly what I want. Unfortunately, I didn't find any docs on this.

**Comment by Pierre-Sassoulas** [comment]
Thanks for the correction :) Where would you expect to find it ? Or alternativelent I think we could make this one a default message as it permits to really clean code when upgrading pylint.

**Comment by stanislavlevin** [comment]
In my opinion, as a default checker for relatively new projects it will be useful, but for old ones, this may be cumbersome at the very first time.
For example, in freeipa case:
```
[builder@localhost freeipa]$ grep -c useless-suppression log
152
```
though all of the occurrences will be fixed.

Probably, I would search for such functionality in
http://pylint.pycqa.org/en/latest/user_guide/message-control.html

**Comment by stanislavlevin** [comment]
Great! Thank you!
